_appmap/test/normalize.py

```diff
@@ -82,6 +86,7 @@ def normalize(dct):
             if len(dct["headers"]) == 0:
                 del dct["headers"]
         if "http_server_request" in dct:
+            dct["http_server_request"].pop("headers", None)
```

Comment:
I'd rather we kept checking for the headers. I assume you changed it because FastAPI formats them differently. If you normalize the capitalization before generating the AppMap like I suggested, you should be able to bring it back.

Reply:
I updated the tests to check that the `http_server_request` properties that depend on header values get set correctly. I can update this so it verifies that `headers` is present. 

Is the combination of those two checks sufficient?

Reply:
OK, though maybe it could use a comment.